src/Test.Plugins/Test.Plugins.csproj

```diff
@@ -6,7 +6,7 @@
   </PropertyGroup>
 
   <PropertyGroup>
-    <TargetFrameworks>netcoreapp2.1;net461</TargetFrameworks>
+    <TargetFrameworks>net461</TargetFrameworks>
```

Comment:
Should the test projects all be netcoreapp2.1, for consistency? The test execution logic was a bit weird, and it seems like tests in VS sometimes run on all framework targets and sometimes just one. If we make all test projects just one target, hopefully this means all tests would always run and would only run once.

Reply:
This isn't a test binary, it is a test plug-in. That is, a plug-in we use for testing. Its build is separate from the rest of the system.

---
In reply to: [391736031](https://github.com/microsoft/sarif-sdk/pull/1812#discussion_r391736031) [](ancestors = 391736031)

Reply:
all test projects now verified to be netcoreapp2.1

---
In reply to: [391844821](https://github.com/microsoft/sarif-sdk/pull/1812#discussion_r391844821) [](ancestors = 391844821,391736031)